lib/resources/r4/related_person.yaml

```diff
@@ -63,3 +63,16 @@ fields:
     - display: Millennium Person Relationships
       system: https://fhir.cerner.com/&lt;EHR source id&gt;/codeSet/40
       info_link: https://fhir.cerner.com/millennium/r4/proprietary-codes-and-systems/#code-set-40-person-relationships
+
+- name: Relationship Level Extension
+  type: CodeableConcept
+  description: Reference to the Encounter associated to the encounter level RelatedPerson.
```

Comment:
I don't think I've ever reviewed the yaml file.  Why aren't you using the Description from the Structure Definition?
Such as The resource's relationship to either the patient or encounter level.
Same comment for the binding description

Reply:
Copied from the wrong structure def! 
Fixed: https://github.com/cerner/fhir.cerner.com/pull/480/commits/98af69a03800c29aa7b196f80be2dd2bd7ab0ee1#diff-261586474bd9faacdccf38c13eea61175a7bba5b7ce8ddb4dea273142eb9f6d6L69